### ISSUE TITLE
Add Pause and resume flow option

### ISSUE THREAD

**Issue by mdshamoon** [glific/glific#1983]
**Is your feature request related to a problem? Please describe.**
[Requirement](https://docs.google.com/document/d/1Q7ozYAHl1wcVQlXcg3iZjQZPHaUQJzQCYHTLaajncrs/edit#)

**Describe the solution you'd like**
Design
https://www.sketch.com/s/ba323b85-11d7-4687-8a19-681a32132bab/a/VgA8Pq
https://www.sketch.com/s/ba323b85-11d7-4687-8a19-681a32132bab/a/Ogb9A2

![image](https://user-images.githubusercontent.com/16714604/134484833-204ab1ce-f7f0-4b3b-a1a6-cfa07656925e.png)
![image](https://user-images.githubusercontent.com/16714604/134484867-6ca51e58-50a9-4d87-91a1-f4eb2a884fed.png)

**Comment by pankaj-ag** [comment]
I don't think this is valid for now. I am moving this task to @glific/glific-frontend-team and they can decide what they want to do.
